0010-async-cloud-team.md

```diff
@@ -0,0 +1,92 @@
+# Asynchronous collaboration for the cloud team
+
+* Status: proposed 
+<!-- {proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)} --> <!-- optional -->
+* Deciders: @TANguyen1893 @mckornfield @emmanue @ZavenArra @peterwwillis @ximena9201
+<!-- Find deciders here: https://github.com/orgs/Greenstand/people  -->
+* Date: 2022-06-07
+
+Technical Story: {description | ticket/issue URL} <!-- optional -->
+
+## Context and Problem Statement
+
+Engineering teams at Greenstand have typically incubated using weekly meetings and semi-synchronous communication in Slack.  As team personnel, process, and technologies mature, the scope of individual tasks becomes smaller.  Additionlly, conveying information primarily through synchronous meetings limits participation from those that cannot regularly attend, or are looking to contribute during unstructured time in their busy schedules. 
+
+The cloud team has reached a point of maturity where adopting asynchronous processes can expand productivity, and needs to select the right suite of approaches to support team members in this effort. 
+
+## Decision Drivers <!-- optional -->
+
+* Inclusion of engineers with varying availability
+* Engagement and retainment of new contributors
+* Productivity
+
+## Considered Options
+
+We will select a mix of options from the below list.
+
+1. ADRs for all proposed or accepted decisions
+2. Manage small scope tickets via a cloud team github project board
+3. Continue weekly meetings
+4. Discontinue weekly meetings, and adopt a longer monthly meeting
+5. Coworking times
+6. Additional options / ideas?
+
```

Comment:
I think ADRs for all proposed or accepted decisions, Manage small scope tickets via a cloud team github project board, Continue weekly meetings and Coworking times, would be great. 
We could write a meeting report, which summarizes the topics that were discussed during the meeting and outline the diffferent tasks which need to be done; this could be linked to a github project board for a holistic view of the tasks.

Reply:
@emmanue great comments.  How should we identify who writes the meeting report? Keeping minutes like this or reporting back after each meeting has been a consistent problem for our teams, due to low time availability.

Reply:
@ZavenArra I'm up for the task. If there is a template on which to start from that would be great. If not I will write one from scratch and use that to write the the reports after every weekly meeting.

Reply:
@emmanue we do not have a meeting minutes template specifically, but there are some easy ones to follow in Evernote that I really like. If using Evernote, when you start a new note, a few options are shown on the page for templates, with "Meeting Note" being one of them. I've attached a screenshot of that template for you to check out. 

![Evernote Meeting Note Template Example](https://user-images.githubusercontent.com/106110299/173136421-3dbe0587-c620-4358-95cc-1760a42c3871.png)

The Greenstand google drive also has nice templates created just for Greenstand. But they just have our logo, footer noting the year, and formatting specs. If you wanted, you could draft a meeting minute template starting with a Greenstand basic template, then I can upload it to that space so that it is held there forever for future use. 

Just let me know if I can help! :)

Reply:
Thanks @tabi-k please could you send me the Greenstand template on slack.